README.md

```diff
@@ -93,6 +93,8 @@ nvm is a version manager for [node.js](https://nodejs.org/en/), designed to be i
 
 ### Install & Update Script
 
+**If you use macOS, you must install Xcode and agree to the Xcode License before you install nvm.**
```

Comment:
this isn't strictly true - it's that you need the things the xcode command line tools install, but you could have installed them other ways. also the license part doesn't seem useful to document here, xcode itself can handle that.